README.md

```diff
@@ -105,6 +105,15 @@ services:
 
 ## Running on Linux
 
+### Arch Linux
+
+```shell
+paru -Sy backrest
```

Comment:
Is paru the standard way to install packages on arch? Where is the backrest package coming from -- I don't publish one as a part of the release process / not sure if it's kept up to date! I'm vaguely aware a few have been published by 3rd parties but I'd like to contact the maintainers and make sure they're staying up before recommending them in the README.md .